scm-ui/ui-types/src/Changesets.ts

```diff
@@ -25,6 +25,7 @@
 import { Collection, Links } from "./hal";
 import { Tag } from "./Tags";
 import { Branch } from "./Branches";
+import { Person } from "@scm-manager/ui-components/src/avatar/Avatar";
```

Comment:
We should not rely on src imports. We should export Person from `@scm-manager/ui-components` or move it to `@scm-manager/ui-types`.